### PR TITLE
About category changes

### PR DESCRIPTION
**Pull request by jonaharagon**
Related to https://discuss.privacyguides.org/t/github-wiki/6207

- [x] Please check this box to confirm you have disclosed any relevant conflicts of interest in your post.
- [x] Please check this box to confirm your agreement to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute your contribution as part of our project.
- [x] Please check this box to confirm you are the sole author of this work, or that any additional authors will also reply to this PR on GitHub confirming their agreement to these terms.

### CODE REVIEW

**Review by freddy-m — APPROVED** [review]
LGTM! Will this exist along with a GitHub wiki, or will that be shelved?

**Review by mfwmyfacewhen — APPROVED** [review]

**Review by blacklight447 — COMMENTED** [review]
It looks great, but wouldn't we want to add profile pictures as well?

**Comment by freddy-m** [comment]
Not sure PFPs are needed, but website links could be cool 👀

**Review by blacklight447 — APPROVED** [review]

File: docs/about/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - [:simple-discourse: Discourse](https://discuss.privacyguides.org/u/freddy)
+    - [:simple-github: GitHub](https://github.com/freddy-m "@freddy-m")
+    - [:simple-mastodon: Mastodon](https://social.lol/@freddy "@freddy@social.lol"){rel=me}
+    - [:fontawesome-solid-envelope: Email](mailto:freddy@privacyguides.org)
+    - [:fontawesome-solid-house: Homepage](https://freddy.omg.lol)
+
+??? person "@mfwmyfacewhen"
+
+    - [:simple-discourse: Discourse](https://discuss.privacyguides.org/u/mfwmyfacewhen)
+    - [:simple-github: GitHub](https://github.com/mfwmyfacewhen "@mfwmyfacewhen")
```
Comment (mfwmyfacewhen) [inline docs/about/index.md:40]:
```suggestion
    - [:simple-github: GitHub](https://github.com/mfwmyfacewhen "@mfwmyfacewhen")
    -[:fontawesome-solid-house: Homepage](https://mfw.omg.lol)
```

**Review by mfwmyfacewhen — CHANGES_REQUESTED** [review]

**Review by mfwmyfacewhen — APPROVED** [review]

**Review by blacklight447 — APPROVED** [review]

File: docs/about/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    - [:simple-discourse: Discourse](https://discuss.privacyguides.org/u/matchboxbananasynergy)
+    - [:simple-github: GitHub](https://github.com/matchboxbananasynergy "@matchboxbananasynergy")
+    - [:simple-mastodon: Mastodon](https://mstdn.social/@matchboxbananasynergy "@matchboxbananasynergy@mstdn.social"){rel=me}
+    - [:fontawesome-solid-envelope: Email](mailto:matchboxbananasynergy@privacyguides.org)
+    - [:fontawesome-solid-house: Homepage](https://banana.omg.lol)
+
+??? person "@kai"
+
+    - [:simple-discourse: Discourse](https://discuss.privacyguides.org/u/Kai)
+    - [:simple-github: GitHub](https://github.com/KaiTebay "@KaiTebay")
```
Comment (KaiTebay) [inline docs/about/index.md:54]:
```suggestion
    - [:simple-github: GitHub](https://github.com/KaiTebay "@KaiTebay")
    - [:fontawesome-solid-house: Homepage](https://kaitebay.com) 
```

**Pull request merged**